src/test/java/org/opensearch/security/dlic/rest/api/RolesMappingApiTest.java

```diff
@@ -422,15 +422,18 @@ public void checkNullElementsInArray() throws Exception{
 		HttpResponse response = rh.executePutRequest(ENDPOINT + "/rolesmapping/opendistro_security_role_starfleet_captains",
 													 body, new Header[0]);
 		Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
+        Assert.assertTrue(response.getBody().contains("`null` is not allowed as json array element"));
```

Comment:
inconsistent indentation

Reply:
Not sure how it happened, opening file in `Intellij` and `sublime` did not show the wrong indentation, however when I open it in `vim`, then I saw the inconsistency, fixed